tools/CheckDates.scala

```diff
@@ -0,0 +1,110 @@
+import sys.process._
+
+object CheckDates {
+  val usage = """
+
+    This tool compares a copyright date in files with the date of last change in git.
+    If the message of last change contains "Documents date fixed" it will be ignored and 
```

Comment:
Suggestion for the way commits are identified as being copyright date updates. Instead of requiring the string "Documents data fixed" somewhere in the commit body adopt a less variable approach by requiring a prefix in the commit subject,

    [Sync copyright date to last commit]

 - It's in the imperative voice
 - The square brackets give the feeling of a category label

An additional benefit of relying only on the subject line is this will halve the number of calls `git log` as a format specifier can be used to include the subject in a string with a parsable format.